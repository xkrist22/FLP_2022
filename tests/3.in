A
a
A
A->aAa
A->a

