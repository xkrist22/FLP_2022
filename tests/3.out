A,a',<Aa>
a
A
A->a'<Aa>
a'->a
<Aa>->Aa'
A->a

